### ISSUE TITLE
[1.0.4] Integrated Circuits not rotating properly

### ISSUE THREAD

**Issue by ReTeYeZ** [amadornes/SCM-Localization-Issues#207]
Integrated Circuits bigger than 2x2 don't rotate with the circuit they are placed on. If you pick the circuit up while rotated different than original state it will crash your game and break random circuits around the area you were standing (turning them invisible).

Here a blueprint to test but caution!
https://gist.github.com/6245f5d19aaedf7b82c9a91868ec8599

**Comment by amadornes** [comment]
Yep. Already known. They're not supposed to rotate because it's pretty much impossible to do so (bear in mind they can be in multiple blocks at the same time if you place them on an edge), though I thought I had fixed this already...
